### PR TITLE
[MM-28140] e2e/system_console/customization: add T1028

### PR DESCRIPTION
**Pull request by mgdelacroix**
#### Summary
Add T1028: Custom brand image and text - true, and uploaded / updated

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28140

### CODE REVIEW

**Review by ashishbhate — APPROVED** [review]

File: e2e/cypress/integration/system_console/ui_and_api/customization_spec.js [hunk 1]
```diff
@@ -396,6 +396,48 @@ describe('Customization', () => {
             expect(config.TeamSettings.EnableCustomBrand).to.equal(false);
         });
     });
+
+    it('T1028 - Custom brand image and text - true, and uploaded / updated', () => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/system_console/ui_and_api/customization_spec.js:400]:
```suggestion
    it('MM-T1028 - Custom brand image and text - true, and uploaded / updated', () => {
```

File: e2e/cypress/support/api/system.js [hunk 2]
```diff
@@ -152,3 +152,16 @@ function uploadLicenseIfNotExist() {
...
+Cypress.Commands.add('apiDeleteBrandImage', () => {
+    return cy.request({
+        headers: {'X-Requested-With': 'XMLHttpRequest'},
+        url: '/api/v4/brand/image',
+        method: 'DELETE',
+        failOnStatusCode: false,
+    }).then((response) => {
+        // both deleted and not existing responses are valid
+        expect(response.status).to.be.oneOf([200, 404]);
+        return cy.wrap({response});
```
Comment (saturninoabril) [inline e2e/cypress/support/api/system.js:165]: As commented in other PR.

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks @mgdelacroix! LGTM, with same comments with other PRs :)

File: e2e/cypress/support/api/system.js [hunk 3]
```diff
@@ -152,3 +152,16 @@ function uploadLicenseIfNotExist() {
...
+        headers: {'X-Requested-With': 'XMLHttpRequest'},
+        url: '/api/v4/brand/image',
+        method: 'DELETE',
+        failOnStatusCode: false,
+    }).then((response) => {
+        // both deleted and not existing responses are valid
+        expect(response.status).to.be.oneOf([200, 404]);
+        return cy.wrap(response);
+    });
+});
```
Comment (saturninoabril) [inline e2e/cypress/support/api/system.js:167]: I missed to mention on my previous review to move this into a separate file `api/brand.js` and also the type at `api/brand.d.ts`. It is to match the folder structure in https://api.mattermost.com.

**Review by saturninoabril — CHANGES_REQUESTED** [review]
@mgdelacroix Test looks good and passed. Last request to move the api command in correct file. Thanks.

File: e2e/cypress/support/api/system.js (hunk 3 above)
Comment (mgdelacroix) [reply e2e/cypress/support/api/system.js]: Done! PTAL :)

**Review by saturninoabril — APPROVED** [review]
Thanks @mgdelacroix, LGTM.

**Review by lindy65 — APPROVED** [review]
Thanks @mgdelacroix - looks good to me 👍

**Pull request merged**